src/c++/perf_analyzer/client_backend/client_backend.h

```diff
@@ -514,4 +515,18 @@ class InferResult {
       size_t* byte_size) const = 0;
 };
 
+struct SslOptionsBase {
+  bool ssl_grpc_use_ssl = false;
+  std::string ssl_grpc_root_certifications_file = "";
+  std::string ssl_grpc_private_key_file = "";
+  std::string ssl_grpc_certificate_chain_file = "";
```

Comment:
Should there be a `ssl_http_use_ssl `

Reply:
My understanding is that HTTP protocol does not need such an option, but gRPC protocol does because it's possible for gRPC to use SSL with all of the other gRPC SSL options not specified. See this conversation:

https://github.com/triton-inference-server/server/pull/3874#discussion_r796938711

@tanmayv25 feel free to clarify 🙏

Reply:
Yes. @jbkyang-nvi, the presence of https:// in the server url  as scheme shows SSL is in use. Hence, no need to have a separate option.